lms/djangoapps/completion/tests/test_services.py

```diff
@@ -72,3 +112,22 @@ def test_completion_service(self):
     def test_enabled_honors_waffle_switch(self, enabled):
         self.override_waffle_switch(enabled)
         self.assertEqual(self.completion_service.completion_tracking_enabled(), enabled)
+
+    def test_vertical_completion(self):
```

Comment:
Can you setup a new test case where some of the components within the vertical is completed. Then make sure the vertical complete value is still `False`?

Reply:
This is now done